scripts/trigger.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -eu
+
+# trigger docker hub
+curl -X "POST" "https://cloud.docker.com/api/build/v1/source/${DOCKER_HUB_TRIGGER_SOURCE}/trigger/${DOCKER_HUB_TRIGGER_ID}/call/" \
```

Comment:
@itskingori I've added a trigger for docker hub. Seems different from the one you used before.

<img width="1089" alt="Screenshot 2019-10-11 at 16 32 40" src="https://user-images.githubusercontent.com/17402708/66660282-3f469680-ec45-11e9-939c-17164354d908.png">

Reply:
Should be fine. Each repo should have it's own unique hook. That said, can't see anything ...

<img width="1136" alt="Screenshot 2019-10-14 at 07 51 55" src="https://user-images.githubusercontent.com/703786/66730012-8ad08e80-ee57-11e9-8e52-cc4bb39f6626.png">

Reply:
Related, https://github.com/Intellection/vault-init/pull/1#issuecomment-541502504.

Reply:
@hadrianvalentine I think you might also have to specify the build rules, something like this ...

<img width="1221" alt="Screenshot 2019-11-06 at 10 27 39" src="https://user-images.githubusercontent.com/703786/68277487-5a5bc900-0080-11ea-854e-e04c329317fd.png">

If you go check now you'll see there are none set and they give you some examples:

<img width="1221" alt="Screenshot 2019-11-06 at 10 30 58" src="https://user-images.githubusercontent.com/703786/68277678-bcb4c980-0080-11ea-80cc-6886c0455f09.png">

It tells Docker Hub what to use for the tag from the version passed in (which must match the regex). For example, in the rules above, it'll create two docker tags, `latest` and `<version>` for each trigger.